### DISCUSSION TITLE
How to keep my activities on GitHub's following section private?

### DISCUSSION THREAD

**Discussion by alidehkhodaei** [community/community#51919] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

 Hello, I would like to know if there is a way to keep my activities such as starring or following other users on GitHub private in the following section without making my account private. Is this possible?

**Comment by radar-blip** [comment, marked as answer]
Not possible at the moment without making everything private.
